improver/wxcode/weather_symbols.py

```diff
@@ -60,6 +60,26 @@
 )
 
 
+def _define_invertible_conditions():
+    """Returns a dictionary of boolean comparator strings where the value is the
+    logical inverse of the key."""
+    invertible_conditions = {
+        ">=": "<",
+        ">": "<=",
+        "OR": "AND",
+        "": "",
```

Comment:
Why this one?

Reply:
The simpler conditions don't have a comparator (AND and OR), so we need the ability to return an empty string too. This could be an if test in the calling function instead.